apps/sap-commerce-cloud/package.json

```diff
@@ -49,10 +48,11 @@
     "@testing-library/user-event": "^12.7.2",
     "@types/jest": "^26.0.20",
     "@types/lodash.get": "^4.4.6",
-    "@types/node": "^14.14.31",
+    "@types/node": "^20.0.0",
```

Comment:
I had to update node in order to support the vite package installation. I picked this one because it looked like the latest stable version but if you're aware of a reason we should be using a different lower version of Node instead I wanted to call that out.

Reply:
being on 14 before is wild...

I think 20 is reasonable (assuming the type versioning is in lock step with node itself, which it probably is)